42.FragmentShaderPathTracer/main.cpp

```diff
@@ -329,7 +336,11 @@ int main()
 			IGPUBuffer::SCreationParams params = {};
 			const size_t size = random.size() * sizeof(uint32_t);
 			params.usage = core::bitflag(asset::IBuffer::EUF_TRANSFER_DST_BIT) | asset::IBuffer::EUF_TRANSFER_SRC_BIT; 
-			buffer = device->createDeviceLocalGPUBufferOnDedMem(params, size);
+			params.size = size;
+			buffer = device->createBuffer(params);
+			auto bufferMemReqs = buffer->getMemoryReqs();
+			bufferMemReqs.memoryTypeBits &= device->getPhysicalDevice()->getHostVisibleMemoryTypeBits();
```

Comment:
should be DeviceLocal because previously used was the function `createDeviceLocalGPUBufferOnDedMem`